### PR TITLE
Fix repository summary on mobile

### PR DESCRIPTION
**Pull request by kolaente**
This PR fixes the repository summary on mobile. Most of it is vertically centering things and some spacing.

#### Before:
Screen Shot 2021-10-15 at 19 12 25

#### After:
Screen Shot 2021-10-15 at 19 28 06

### CODE REVIEW

File: web_src/less/_repository.less [hunk 1]
```diff
@@ -2398,6 +2401,10 @@
       border-radius: 0;
       user-select: none;
 
+      @media @mediaSm {
+        display: none;
+      }
```
Comment (wxiaoguang) [inline web_src/less/_repository.less:2416]: We can just remove the height of `.repository-summary-language-stats`, then the language stats bar can be shown correctly.

File: web_src/less/_repository.less [hunk 2]
```diff
@@ -3085,6 +3092,10 @@ td.blob-excerpt {
 .repository-summary-language-stats {
   height: 48px;
```
Comment (wxiaoguang) [inline web_src/less/_repository.less:3107]: Here, we can remove this, then the layout is still correct I think.

Comment (kolaente) [reply web_src/less/_repository.less:3107]: I tried that, but the height is set to that magic number for a reason. The comment above that line says `prevent page shaking on language bar click` and that's the case when removing it.

**Review by wxiaoguang — APPROVED** [review]

**Review by zeripath — APPROVED** [review]

**Comment by zeripath** [comment]
make lgtm work

**Pull request merged**